crates/js_api/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "rain_orderbook_js_api"
+description = "Javascript bindings for common functions accross the app."
+version.workspace = true
+edition.workspace = true
+license.workspace = true
+homepage.workspace = true
+
+[lib]
+crate-type = ["cdylib"]
+
+[target.'cfg(target_family = "wasm")'.dependencies]
```

Comment:
we dont need cfg at all in this crate, everything will be without cfg, so normal `dependencies`